scipy/stats/_stats.pyx

```diff
@@ -30,7 +30,8 @@ cdef double von_mises_cdf_series(double k, double x, unsigned int p):
 
 
 cdef von_mises_cdf_normalapprox(k, x):
-    b = math.sqrt(2 / PI) / scipy.special.i0e(k) # Check for negative k
+    # 0.79788... is sqrt(2/PI).
+    b = 0.79788456080286535588 / scipy.special.i0e(k)  # Check for negative k
```

Comment:
I think Cython docs recommend using an anonymous `enum` for declaring constants, though it likely doesn't matter much.

Reply:
`enum`s only allow integer values:

```
Error compiling Cython file:
------------------------------------------------------------
...
cdef enum:
    myconst = 123.456
             ^
------------------------------------------------------------

hello.pyx:2:14: Cannot assign type 'double' to 'int'
```

Reply:
You could also use a `DEF`, compare to e.g.

https://github.com/scipy/scipy/blob/master/scipy/special/_loggamma.pxd#L34

Reply:
I updated the code to use a DEF statement.  I'm not sure it is actually better this way, but I'm OK with it.